buildSrc/src/main/java/Config.kt

```diff
@@ -0,0 +1,8 @@
+object Version {
```

Comment:
Why do this instead of just defining `ext.compileSdkVersion` etc in the base `build.gradle`? We don't really need to pull in the kotlin dsl.